pytest/lib/cluster.py

```diff
@@ -807,13 +828,13 @@ def spin_up_node(config,
     return node
 
 
-def init_cluster(num_nodes,
-                 num_observers,
-                 num_shards,
-                 config,
-                 genesis_config_changes,
-                 client_config_changes,
-                 prefix="test"):
+def init_cluster(num_nodes: int,
+                 num_observers: int,
+                 num_shards: int,
+                 config: Config,
+                 genesis_config_changes: GenesisConfigChanges,
+                 client_config_changes: ClientConfigChanges,
+                 prefix="test") -> typing.Tuple[str, typing.List[str]]:
```

Comment:
mini nit: Personally I like to add a trailing comma right before the closing paren to force each arg to a new line. Up to you since it's not an established convention.

Reply:
good to know, thanks, that looks nicer: 7af6bf4ab